aioairq/__init__.py

```diff
@@ -5,7 +5,14 @@
 __url__ = "https://www.air-q.com"
 __license__ = "Apache License 2.0"
 __version__ = "0.3.2"
```

Comment:
Let's bump the version. Until 1.0.0 is reached I try to vaguely follow the `0.<breaking>.<feature or fix>` convention 🤷
```suggestion
__version__ = "0.3.3"
```

Reply:
See https://github.com/CorantGmbH/aioairq/pull/4#discussion_r1488516679.

I will use 0.4.0 then, instead of 0.3.3.